services/workflows-service/src/alert/alert.service.intg.test.ts

```diff
@@ -117,6 +117,8 @@ describe('AlertService', () => {
               enabled: true,
             },
             project,
+            undefined,
+            { crossEnvKey: 'TEST' },
```

Comment:
Can you add a test that verify the crossEnv creation for a project ?

with default value from project name and with explicit key?

Reply:
WDYM by creation for a project? didnt get the test